### DISCUSSION TITLE
When 1 chat individually to a person ,the edit options is become similar to a Channel . Why there is no difference between a normal chat and Channel ?

### DISCUSSION THREAD

**Discussion by anjuarevero** [GetStream/stream-chat-android#2369] [Q&A]
**Describe the bug**
A clear and concise description of what the bug is

I send a direct message , For me it showing View info, Leave channel, Add member , Delete Member etc .So what is the difference between a channel and Direct message ?
Normal direct message chat options is entirely different from a channel right ?
Referred WhatsApp, Fb messenger, Teams etc .
Direct chat should have chat features like Hide chat , delete chat etc and Channel features should have their own.

**SDK version**
- x.x.x

**To Reproduce**

**Expected behavior**

**Device:**
 - Vendor and model: [e.g. Samsung S8]
 - Android version: [e.g. 9]

**Screenshots**

**Comment by adasiewiczr** [comment]
Hey @anjuarevero,
the main difference between a regular channel (created based on id) and DM (created based on members list) is that the second one is unique and the same group of users has only one channel. You can read more about this [here](https://getstream.io/chat/docs/react/query_users/?language=kotlin).
In our default implementation, options are similar (with a few exceptions) and they are also based on [user permissions](https://getstream.io/chat/docs/react/chat_permission_policies/?language=kotlin).
You can also customize available options and provide custom implementation with different options using `ChannelListView::setMoreOptionsClickListener`

**Reply by Asep3466** [reply]
Masukan Telepon
